index.html

```diff
@@ -12340,7 +12454,7 @@ <h3>States and Properties</h3>
 				</ul>
 				</li>
 				<li>Otherwise, ignore the value and treat the property as not present.</li>
-			</ul>
+			</ul>	
 			<p class="note">In <abbr title="User Interface Automation">UIA</abbr>, the user agent might leave the corresponding property set to &quot;unsupported.&quot;</p>
```

Comment:
Change "property"
to "attribute".
(for consistency with other points in this list).

Reply:
happy to accept a PR cleaning these up but I don't want to make this any larger than it has to be.

Reply:
Resolving conversation as mentioned in [previous comment](https://github.com/w3c/aria/pull/985#discussion_r296823383).